modules/oe_subscriptions_anonymous/tests/src/Trait/MessageTemplateTrait.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_subscriptions_anonymous\Trait;
+
+/**
+ * Trait to assert messages templates.
+ */
+trait MessageTemplateTrait {
+
+  /**
+   * Asserts that the confirm message is present.
+   *
+   * @return bool
+   *   If the message is present in the page.
+   */
+  public function confirMessageExists(): bool {
```

Comment:
1. Methods that assert stuff (see first line of the phpdoc) should start with `assert`.
2. Method should be protected.
3. Parameters should not be hardcoded, we should pass an array of values.
4. Method should be called assertHtmlStatusMessage()
5. The message type should be passed too.

Reply:
Fixed different issues, like visibility, name and parameters